### PR TITLE
add custom data finder

### PR DESCRIPTION
**Pull request by grburgess**
this addresses #67 by adding a data finder

### CODE REVIEW

**Comment by sbailey** [comment]
@dkirkby do you have the bandwidth to take a look at this?

**Comment by dkirkby** [comment]
Yes, I have it on my list now.

File: speclite/filters.py [hunk 1]
```diff
@@ -242,6 +242,7 @@
 import astropy.table
 import astropy.units
 import astropy.utils.data
```
Comment (dkirkby) [inline speclite/filters.py:244]: Any reason not to remove this import now?

**Review by mcbeth — APPROVED** [review]
Thanks for doing this

File: speclite/filters.py (hunk 1 above)
Comment (grburgess) [reply speclite/filters.py]: Nope! It is removed now.

**Comment by grburgess** [comment]
@dkirkby @mcbeth thanks for looking so quickly .

**Pull request merged**